parts/linux/cloud-init/artifacts/bootstrap.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-set -euxo pipefail
-# TODO: propagate exit status to bootstrapper
-if [ ! -f /opt/azure/containers/nbc.json ]; then 
-    exit 1
-fi
-mkdir -p /var/log/azure/aks
-/opt/azure/containers/nbcparser --bootstrap-config=/opt/azure/containers/nbc.json > /var/log/azure/aks/bootstrap.log 2>&1
```

Comment:
Are we still logging nbcparser/node-bootstrapper process somewhere if we remove this logging command?

Reply:
path to log file is hard-coded in node-bootstrapper. (questionable, but I don't want user to be able to change the path yet, we may want to ways to export it)

stdout/stderr currently replicating output of CSE scripts and doesn't match logging.